tests/test_query_parsing.py

```diff
@@ -144,6 +144,38 @@ async def test_nested_query(self):
     self._test_query(query_str5, config, normalized_query_str5, correct_fp, correct_service, correct_tables, 3)
 
 
+    query_str6 = '''
```

Comment:
Doesn't have to be for this PR but can we do something like turning this into a dict? Seems a bit easier